setup.py

```diff
@@ -44,7 +44,7 @@ def read(f):
       maintainer_email='aio-libs@googlegroups.com',
       url='https://github.com/aio-libs/aiomcache/',
       license='BSD',
-      packages=find_packages(),
+      packages=find_packages(exclude=["tests", "tests.*"]),
```

Comment:
We should just make this explicit as per our other repos:

```suggestion
      packages=("aiomcache",),
```